test/src/test/java/jenkins/cli/StopBuildsCommandTest.java

```diff
@@ -57,20 +56,20 @@
     @Test
     public void shouldStopLastBuild() throws Exception {
         final FreeStyleProject project = createLongRunningProject(TEST_JOB_NAME);
-        project.scheduleBuild2(0).waitForStart();
+        FreeStyleBuild build = project.scheduleBuild2(0).waitForStart();
+        j.waitForMessage("Sleeping", build);
```

Comment:
Here and elsewhere, we need to wait for the build to have started before we try to stop it. Otherwise the test can flake because it wouldn't have stopped what the test was expecting to stop.